### PR TITLE
Add CCM 0.10.0 release notes

### PR DESCRIPTION
**Pull request by vexx32**
## Description Of Changes

Added release notes for CCM 0.10.0 release

## Motivation and Context

new release, new notes :3

## Testing

* [x] I have previewed these changes using the [Docker Container](https://github.com/chocolatey/docs/tree/master/.devcontainer) or another method before submitting this pull request.

## Change Types Made

* [ ] Minor documentation fix (typos etc.).
* [x] Major documentation change (refactoring, reformatting or adding documentation to existing page).
* [ ] New documentation page added.
* [ ] The change I have made should have a video added, and I have raised an issue for this.
    * Issue #

## Change Checklist

* [ ] Requires a change to menu structure (top or left hand side)/
* [ ] Menu structure has been updated

## Related Issue

N/A

### CODE REVIEW

File: input/en-us/central-management/release-notes.md [hunk 1]
```diff
@@ -29,6 +29,43 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
 - Please see https://github.com/chocolatey/chocolatey-licensed-issues/labels/CentralManagement
 - Some issues may be held internally, please follow your support routes to learn more.
 
+## 0.10.0 (August 30, 2022)
+
+### Features
+
+- Implement Recurring Deployments
```
Comment (pauby) [inline input/en-us/central-management/release-notes.md:36]:
```suggestion
- Added recurring Deployments.
```

File: input/en-us/central-management/release-notes.md [hunk 2]
```diff
@@ -29,6 +29,43 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+- Allow users to Duplicate an existing deployment
```
Comment (pauby) [inline input/en-us/central-management/release-notes.md:37]:
```suggestion
- Add ability to duplicate an existing Deployment.
```

File: input/en-us/central-management/release-notes.md [hunk 3]
```diff
@@ -29,6 +29,43 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+- Retention Policies - Automatically delete a computer that hasn't reported into CCM for a configurable period of time
+- Implement a dark/light mode switcher
+- API - Add a way to query licensed machine count - see [Licensed #272](https://github.com/chocolatey/chocolatey-licensed-issues/issues/272)
```
Comment (pauby) [inline input/en-us/central-management/release-notes.md:40]:
```suggestion
- Retention Policies - Automatically delete a computer that hasn't reported in for a configurable period of time. This defaults to 365 days.
- Implement a dark/light mode.
- API - Add a way to query licensed machine count - see [Licensed #272](https://github.com/chocolatey/chocolatey-licensed-issues/issues/272).
```

File: input/en-us/central-management/release-notes.md [hunk 4]
```diff
@@ -29,6 +29,43 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+### Bug Fixes
+
+- Some auditing data is lost in the database when some entities are updated
+  - Previously, editing a Group, Software, or Computer entity would erase the `CreatorUserId` and `CreationTime` auditing fields
+  - As part of this fix, entities with a `NULL` `CreatorUserId` will copy the `LastModifierUserId` into their `CreatorUserId` when they are next updated
+- Retrieving a deployment from the GetDeploymentPlanForView method via the API causes a JsonSerializationException
+- Notifications table shows wrong creation time
+- The "Generated" time on Excel and PDF Reports is in UTC regardless of the specified Time Zone
+- Login Attempts section is broken for non-administrative users
+- API - Adding direct ComputerGroup / GroupGroup mappings via the API does not update group counts or deployment eligibility immediately
```
Comment (pauby) [inline input/en-us/central-management/release-notes.md:51]:
```suggestion
- Fixed issue where auditing data may be lost in the database when some entries are updated. 
  - Editing a Group, Software, or Computer would erase the user who created it and the time it was created. As part of this fix, entries without that information will be set to the user who last modified it and the time it was last modified.
- API - Exception when retrieving a deployment from the `GetDeploymentPlanForView` method.
- The creation time on the Notifications table was incorrect.
- Excel and PDF Reports incorrect show the time in UTC and not the local time zone.
- Non-administrative users cannot view the Login Attempts section.
- API - Group statistics were not updated when adding Computers / Groups via the API.
```

**Review by pauby — CHANGES_REQUESTED** [review]

File: input/en-us/central-management/release-notes.md [hunk 5]
```diff
@@ -29,6 +29,43 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+- Add visual indicators that editing/adding/removing deployment steps is disabled when the deployment schedule is outdated
+- Provide ability to specify basic parameters on Basic step
+- Add email notifications when a scheduled deployment fails to start
+- Warn Users when their Chocolatey License Expiration is coming up
+- Remember selected value for Show Entries dropdown on tables
+- Replace loading gif used on datatables
+- CCM User passwords must be at least 6 characters long
+- Enable 2FA as part of authentication - see [Licensed #300](https://github.com/chocolatey/chocolatey-licensed-issues/issues/300)
+- Uninstalling chocolatey-management-web should remove IIS Website and Application Pool
+- Add additional auditing fields for deployment plans
```
[the remark this answers is not in the document]
Comment (pauby) [reply input/en-us/central-management/release-notes.md:67]: I know these lines relate to an internal / external issue. We will need to update the issues title to reflect these changes.

File: input/en-us/central-management/release-notes.md (hunk 4 above)
Comment (vexx32) [reply input/en-us/central-management/release-notes.md:51]: Noting here that we don't update the creation date as far as I am aware. I believe Gary mentioned that while using the last modified user has _some_ chance of being decently correct or relevant, the creation time is practically guaranteed to always be completely incorrect if we did the same for it, so we left it empty.

```suggestion
- Fixed issue where auditing data may be lost in the database when some entries are updated. 
	  - Editing a Group, Software, or Computer would erase the user who created it and the time it was created. As part of this fix, entries without that information will set the creating user to the user who last modified it.
	- API - Exception when retrieving a deployment from the `GetDeploymentPlanForView` method.
	- The creation time on the Notifications table was incorrect.
	- Excel and PDF Reports incorrect show the time in UTC and not the local time zone.
	- Non-administrative users cannot view the Login Attempts section.
	- API - Group statistics were not updated when adding Computers / Groups via the API.
```

File: input/en-us/central-management/release-notes.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (vexx32) [reply input/en-us/central-management/release-notes.md:67]:
```suggestion- Computers tab should display Group enrolment - see [Licensed #223](https://github.com/chocolatey/chocolatey-licensed-issues/issues/223).
- Creating a Deployment and then clicking Cancel without adding steps or saving once should remove the deployment entirely.
- Add the Deployment name to its Step Details pages.
- Add visual indicators that editing/adding/removing deployment steps is disabled when the deployment schedule is outdated.
- Add option of `--version` and `--pre` for a Basic Deployment Step.
- Send email notifications when a scheduled deployment fails to start.
- Warn Users when their Chocolatey License Expiration is coming up
- Remember the specified value for Show Entries dropdown on tables.
- Replace loading animation.
- Require new user passwords to be at least 6 characters long.
- Enable email 2FA as part of authentication - see [Licensed #300](https://github.com/chocolatey/chocolatey-licensed-issues/issues/300).
- Uninstalling chocolatey-management-web should remove IIS Website and Application Pool.
- Add additional auditing fields for Deployments.
```

Reading back through these suggestions makes me remember that "deployment plans" is kind of an internal-only name; all our docs just refers to them as "deployments"... I think it's best we stick to that (and remember for internal issue names in future 😭 ). Updated the suggestion here from yours to match.

File: input/en-us/central-management/release-notes.md [hunk 6]
```diff
@@ -29,6 +29,42 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+- Implement a dark/light mode.
+- API - Add a way to query licensed machine count - see [Licensed #272](https://github.com/chocolatey/chocolatey-licensed-issues/issues/272).
+
+### Bug Fixes
+
+- Fixed issue where auditing data may be lost in the database when some entries are updated. 
+  - Editing a Group, Software, or Computer would erase the user who created it and the time it was created. As part of this fix, entries without that information will set the creating user to the user who last modified it.
+- API - Exception when retrieving a deployment from the `GetDeploymentPlanForView` method.
+- The creation time on the Notifications table was incorrect.
+- Excel and PDF Reports incorrect show the time in UTC and not the local time zone.
```
Comment (vexx32) [inline input/en-us/central-management/release-notes.md:48]:
```suggestion
- Excel and PDF Reports incorrecly show the time in UTC and not the local time zone.
```

File: input/en-us/central-management/release-notes.md [hunk 7]
```diff
@@ -29,6 +29,42 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+
+### Bug Fixes
+
+- Fixed issue where auditing data may be lost in the database when some entries are updated. 
+  - Editing a Group, Software, or Computer would erase the user who created it and the time it was created. As part of this fix, entries without that information will set the creating user to the user who last modified it.
+- API - Exception when retrieving a deployment from the `GetDeploymentPlanForView` method.
+- The creation time on the Notifications table was incorrect.
+- Excel and PDF Reports incorrectly show the time in UTC and not the local time zone.
+- Non-administrative users cannot view the Login Attempts section.
+- API - Group statistics were not updated when adding Computers / Groups via the API.
```
Comment (vexx32) [inline input/en-us/central-management/release-notes.md:50]: Think this needs to be a bit more specific, the main Computers/Groups API was fine, it was directly modifying the links that was busted

```suggestion
- API - Group statistics were not updated when adding ComputerGroups / GroupGroups via the API.
```

File: input/en-us/central-management/release-notes.md [hunk 8]
```diff
@@ -29,6 +29,42 @@ This covers the release notes for the Chocolatey Central Management (`chocolatey
...
+- Creating a Deployment and then clicking Cancel without adding steps or saving once should remove the deployment entirely.
+- Add the Deployment name to its Step Details pages.
+- Add visual indicators that editing/adding/removing deployment steps is disabled when the deployment schedule is outdated.
+- Add option of `--version` and `--pre` for a Basic Deployment Step.
+- Send email notifications when a scheduled deployment fails to start.
+- Warn Users when their Chocolatey License Expiration is coming up
+- Remember the specified value for Show Entries dropdown on tables.
+- Replace loading animation.
+- Require new user passwords to be at least 6 characters long.
+- Enable email 2FA as part of authentication - see [Licensed #300](https://github.com/chocolatey/chocolatey-licensed-issues/issues/300).
```
Comment (vexx32) [inline input/en-us/central-management/release-notes.md:64]:
```suggestion
- Add option for email 2FA as part of authentication - see [Licensed #300](https://github.com/chocolatey/chocolatey-licensed-issues/issues/300).
```

**Review by pauby — APPROVED** [review]

**Pull request merged**